src/main/static/src/components/kutsutut/KutsututTable.js

```diff
@@ -79,6 +82,7 @@ class KutsututTable extends React.Component<Props, State> {
                        fetchMoreAction: this.onSubmitWithoutClear.bind(this),
                    }}
                    tableLoading={this.props.isLoading}
+                   subComponent={(row: any) => <KutsuDetails kutsu={this.props.kutsus.find(kutsu => kutsu.id === row.original.id)} L={this.props.L} locale={this.props.locale} />}
```

Comment:
mä en jaksa nillittää tästä mutta tää on se tyypityksen tyhmyys mun mielestä. Mitä järkee että sitä tarvitaan, jos se kirjoitetaan any:na sisään. Mua ärsyttää tässä kohtaa aina ne ts virheet ihan suunnattomasti. Ei tartte siis korjata mitään kuhan vaan aivoilen.

Reply:
and thats the problem.. dunno whether react-table typings are available in _flow_. If we were using ts one could just grab missing type definitions from https://www.npmjs.com/package/@types/react-table